src/app/pages/sub/sub.page.html

```diff
@@ -0,0 +1,20 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-button (click)="dismissModalSub()"><ion-icon name="arrow-back-outline"></ion-icon></ion-button>
+    </ion-buttons>
+    <ion-title>{{this.subscription ? 'Abonnements de ' + user.firstName : 'Abonnés de ' + user.firstName}}</ion-title>
```

Comment:
`{{this.subscription ? 'Abonnements' :  'Abonnés'}} de ' {{user.firstName}}` suffit. Je l'avais déjà corrigé dans le PR d'avant.